cleaning-service.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8" lang="en">
+        <link rel="stylesheet" href="styles.css">
+        <script src="scripts.js"></script>
+        <title>Cleaning Service</title>
+    </head>
+    <body>
+        <div class="container">
+            <header>
+                <nav class="navbar">
+                    <a href=#About-us><div class="navbaritem">ABOUT US</div></a>
+                    <a href="Services"><div class="navbaritem">SERVICES</div></a>
+                    <a href="Blog"><div class="navbaritem">BLOG</div></a>
+                    <a href="FAQ"><div class="navbaritem">FAQ</div></a>
+                    <div class="navbaritem"><button id="LOGIN"><b>LOGIN</b></button></div>

```

Comment:
If class name have a long word , 2 words perhaps you need to add dash into class name like this : navbar-item